src/App.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ] As requested by the previous reviewer, kindly remove the `disable-linters` as the rules help us to write industry-standard code